src/com/oltpbenchmark/util/PlacementPolicy.java

```diff
@@ -0,0 +1,27 @@
+package com.oltpbenchmark.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+ * For GeoPartitioned clusters, the configuration specifying how the
+ * tables should be placed is a list of cloud.region.zone.min_number_of_replicas
+ * blocks in addition to the total replication factor.
+ * 
+ * This class encapsulates such a placement policy. This class is used with Gson to 
+ * generate a JSON to use for tablespace creation. The field names should not be
+ * changed as they should correspond exactly with the json keys expected by the SQL syntax.
+ */
+ 
+public class PlacementPolicy {
+    private int num_replicas;
+    private List<PlacementBlock> placement_blocks = new ArrayList<PlacementBlock>();
+
+    public void setNumReplicas(int numReplicas) {
```

Comment:
It looks like it is possible to initialize this field in constructor. Setter can be removed.

Reply:
Done